README.md

```diff
@@ -99,3 +99,7 @@ To provide that support the API versions of objects should be those that work fo
 ## Status of the Project
 
 This project is still under active development, so you might run into [issues](https://github.com/helm/charts/issues). If you do, please don't be shy about letting us know, or better yet, contribute a fix or feature.
+
+## Happy helming in China
+
+If you are in China, there are some problems to use Helm Charts directly (e.g. gcr.io, googleapis.com etc), you can use this mirror repo at https://github.com/cloudnativeapp/charts.
```

Comment:
maybe:

If you are in China, there are some problems to use upstream Helm charts (e.g. images hosted on gcr.io, quary.io, and charts hosted on googleapis.com etc), we recommend you to use the "localized version" repo at https://github.com/cloudnativeapp/charts, which periodically sync charts from here and automatically replace unavailable URL by local mirror in every chart.